akka-bench-jmh/README.md

```diff
@@ -0,0 +1,10 @@
+# Akka Microbenchmarks
+
+This subproject contains some microbenchmarks excercising key parts of Akka.
+
+You can run them like:
+
+   project akka-bench-jmh
+   jmh:run -i 3 -wi 3 -f 1 .*actor.*
```

Comment:
I think you almost always run one specific class, so better to write out a full class name in the example.